test/metabase/native_query_analyzer/replacement_test.clj

```diff
@@ -56,12 +56,25 @@
                                      "city"   "town"}  ; make sure FFs aren't replaced
                            :tables  {"people" "folk"}})))))
 
-(deftest ^:parallel referenced-card-test
+(deftest referenced-card-test
```

Comment:
Since this uses `with-temp` I think it's not `parallel`-safe?

Reply:
I'd expect that our linter would be configured to pick up such a common helper if that's the case.

Since it's a whole new table on the side, I'd expect it to be thread-safe? The only issues I can think of are when it's picked up by sync etc, which I think we handle now.

If this is the case, shouldn't we mark it explicitly with `:synchronized` then?

If we're just not sure, perhaps it's better to just leave it until we see a concrete flake?